### ISSUE TITLE
Suggestion: make launchers more common for random equipment loadouts

### ISSUE THREAD

**Issue by mitchdshawn** [SkaceKamen/random-infantry-skirmish#16]
Currently vehicles have an excessive advantange.  If a team starting rolling with a few, and the other team can't get any more kills for money, they round is pretty much over.  Increasing the probability of a soldier spawning with a launcher would greatly help, and give the losing team a bit more of a chance to come back into the game.

**Comment by SkaceKamen** [comment]
Yes, I thought about that. The main issue is, that the AI for AT infantry is very weak. I tried equipping everyone with AT launchers and they still weren't able to kill anything.
The other issue is, when there's big number of AA launchers in the game, playing anything air-related becomes impossible, because AI for AA infantry is actually pretty ruthless.
Right now, the best solution is to use custom loadout and equip yourself with AT launcher yourself.
